docs/README.md

```diff
@@ -1,540 +1,16 @@
-
-### How to recreate the gh-pages branch
-To recreate the gh-pages branch, do the following:
-- checkout the gh-pages branch to your local system
-```shell
-git clone -b gh-pages {{ config.repo_url }} KubeStellar
-cd KubeStellar
-```
-- delete all files in the branch and push it to GitHub
-```shell
-rm -rf *
-git add; git commit -m "removing all gh-pages files"; git push -u origin gh-pages
-```
-- switch to the 'release' branch, switch to /docs and run 'mike deploy' for the release branch. Add the alias 'latest' for the latest release 
-```shell
-git checkout release-0.22
-git pull
-mike deploy --push --rebase --update-aliases release-0.22 latest
-```
-- switch back to the gh-pages branch and recreate the home.html, index.html, and CNAME files as needed (make sure you back out of the docs path first before switching to gh-pages because that path does not exist in that branch)
-```shell
-cd ..
-git checkout gh-pages
-git pull
-vi index.html
-vi home.html
-vi CNAME
-```
-- push the new files into gh-pages
-```shell
-git add .;git commit -m "add index, home, and CNAME files";git push -u origin gh-pages
-```
-- go into the GitHub UI and go to the settings for the project and click on 'Pages' to add https://docs.kubestellar.io as the domain and check the box to enforce https.
-
-- if the above did not work, then you might have an issue with the GoDaddy domain (expired, files missing, etc.)
-
-### How to delete a rendering of a branch
-
-Use `mike delete $branch_name`, either acting locally on your checked out `gh-pages` branch (after pull and before git commit and push) or acting more directly on the remote repo using `--remote` and `--push`. See [the mike delete command doc](https://github.com/jimporter/mike?tab=readme-ov-file#deleting-docs).
-
-## Publishing Workflow
-
-All documentation building and publishing is done using GitHub Actions in
-`.github/workflows/docs-gen-and-push.yml`. This workflow is triggered either manually or by a push to a branch named `main` or `release-<something>` or `doc-<something>`. This workflow will actually do something _ONLY_ if either (a) it is acting on the shared GitHub repository at `github.com/kubestellar/kubestellar` and on behalf of the repository owner or (b) it is acting on a contributor's fork of that repo and on behalf of that same contributor. The published site appears at `https://pages.github.io/kubestellar/${branch}` in case (a) and at `https://${repo_owner}.github.io/${fork_name}/${branch}` in case (b). This workflow will build and publish a website _version_ whose name is the same as the name of the branch that it is working on. This workflow will also update the relevant `mike` alias, if necessary.
+Please see: **[Documentation Management Overview](../contribution-guidelines/operations/document-management.md)**
```

Comment:
Remember that the doc source files are dual use (https://docs.kubestellar.io/unreleased-development/contribution-guidelines/operations/document-management/#dual-use-documentation-sources). This link does not work in both views. See https://github.com/kubestellar/kubestellar/blob/main/CONTRIBUTING.md?plain=1 for an example of coping.